src/yadg/parsers/xpstrace/phispe.py

```diff
@@ -0,0 +1,438 @@
+                "name": trace_def[2],
+                "atomic_number": trace_def[3],
+                "num_datapoints": int(trace_def[4]),
+                "step": float(trace_def[5]),
+                "start": float(trace_def[6]),
+                "stop": float(trace_def[7]),
+                "dwell_time": trace_def[10],
+                "e_pass": trace_def[11],
+                "description": trace_def[12],
+            }
+        )
+    return trace_defs
+
+
+def _process_traces(spe: list[bytes], trace_defs: list[dict]) -> dict:
+    """Processes the spectral traces in the file.
+
+    Parameters
+    ----------
+    spe
+        The lines of bytes read from file.
+
+    trace_defs
+        The list of trace definitions parsed from the file header.
+
+    Returns
+    -------
+    dict
+        A dictionary containing the binding energies constructed from
+        the trace definitions and the corrresponding XPS traces.
+
+    """
+    data = b"".join(spe[spe.index(b"EOFH\n") + 1 :])
+    data_header = _read_value(data, 0x0000, data_header_dtype)
+    assert data_header["num_traces"] == len(trace_defs)
+    # All trace headers I have seen are 192 (0xc0) bytes long.
+    assert data_header["trace_header_size"] / trace_header_dtype.itemsize == len(
+        trace_defs
+    )
+    assert data_header["data_header_size"] == data_header_dtype.itemsize
+    trace_headers = _read_values(data, 0x0010, trace_header_dtype, len(trace_defs))
+    traces = {}
+    for trace_header, trace_def in zip(trace_headers, trace_defs):
+        assert trace_header["trace_number"] == trace_def["trace_number"]
+        assert trace_header["num_datapoints"] == trace_def["num_datapoints"]
+        # Contruct the binding energies from trace_def.
+        energies, dE = np.linspace(
+            trace_def["start"],
+            trace_def["stop"],
+            trace_def["num_datapoints"],
+            endpoint=True,
+            retstep=True,
+        )
+        E = {"n": energies.tolist(), "s": [abs(dE)] * len(energies), "u": "eV"}
+        # Construct data from trace_header
+        data_dtype = np.dtype(f'{trace_header["data_dtype"].decode()}')
+        data_offset = trace_header["end_of_data"] - trace_header["num_data_bytes"]
+        datapoints = _read_values(
+            data, data_offset, data_dtype, trace_header["num_datapoints"]
+        )
+        dwell_time = _read_value(data, trace_header["end_of_data"], "<f4")
+        np.testing.assert_almost_equal(dwell_time, float(trace_def["dwell_time"]))
+        # TODO: Figure out the correct error. This signal count should
+        # somehow be a Poisson distribution, i.e. the error should be
+        # something like s ~ sqrt(n). The counts per second only seem to
+        # be taking values in steps of 12.5cps, so taking this as error.
+        y = {
+            "n": datapoints,
+            "s": [12.5] * len(datapoints),
+            "u": trace_header["y_unit"].decode().strip(),
```

Comment:
So, the unit in the json file is `c/s` which pint interprets as:
```
>>> ureg("c/s")
<Quantity(1.0, 'speed_of_light / second')>
```

That's not good. Either we coerce the unit here to something pint understands, or we add another unit overwrite in `pintutils.py`. I'd rather have it done here. Two options that work:
```
>>> ureg("count/s") 
<Quantity(1.0, 'count / second')>
>>> ureg("cps")     
<Quantity(1, 'counts_per_second')>
```